### ISSUE TITLE
doesn't work on chromebook

### ISSUE THREAD

**Issue by jgallen23** [siadat/chrome-ff#1]
using the chromebook search button:
![screenshot 2016-01-20 at 9 07 31 am - display 1](https://cloud.githubusercontent.com/assets/27936/12456588/56821124-bf55-11e5-8d2a-78bd044ecc9c.png)

takes me here:
![screenshot 2016-01-20 at 9 07 56 am](https://cloud.githubusercontent.com/assets/27936/12456602/65c0f9fc-bf55-11e5-88e6-8ea140a39e95.png)

Here's the full url: chrome-extension://gehejppllbljofcafobomimemelelbce/?q=https%3A%2F%2Fgithub.com%2Fsiadat%2Fchrome-ff%2Fissues%2Fnew%236229.6576

**Comment by siadat** [comment]
@jgallen23 thank you for reporting this problem :+1: 

I will let you know when it is fixed.

**Comment by jgallen23** [comment]
I loaded up the extension locally to see if I could fix it, but it doesn't look like onInputEntered is getting called.  I tried another extension that does something similar and got the same results.  I've added a bug for the chromium team to look at: https://code.google.com/p/chromium/issues/detail?id=580780

**Comment by siadat** [comment]
Exactly. I don't have a Chromebook, but I was able to reproduce the problem in the App Launcher on OSX. Even pressing a key triggers a different set of events. I have started working on a patch, I might submit it if the Chromium team is interested in fixing the issue.

**Comment by siadat** [comment]
I couldn't work on my patch idea for Chromium, because I have setup and built Chromium on Ubuntu, and there doesn't seem to be a Google App Launcher on Linux to reproduce the problem there.
